lib/cmds/init/success.ts

```diff
@@ -0,0 +1,68 @@
+import { highlight } from 'cli-highlight'
```

Comment:
I'd like to see `chalk` imported here :) we neeeeeed colors to differentiate between output sections!
Try not to over do it though, let's keep it subtle. 🙏🏼 ❤️